studio/src/fields/secure-text-field.js

```diff
@@ -0,0 +1,86 @@
+import { html, LitElement } from 'lit';
+import { EVENT_INPUT } from '../constants.js';
+
+export class SecureTextField extends LitElement {
+    static properties = {
+        id: { type: String },
+        label: { type: String },
+        value: { type: String },
+        isEditable: { type: Boolean, state: true },
+        showSecureTextField: { type: Boolean, state: true }
+    };
+
+    constructor() {
+        super();
+        this.id = '';
+        this.label = '';
+        this.value = '';
+        this.disabled = false;
+        this.isEditable = false;
+        this.showSecureTextField = true;
+    }
+
+    updated(changedProperties) {
+        if (changedProperties.has('value')) {
+            if (!this.value || this.value === '') {
```

Comment:
We can shorten this to if (!this.value) {...}, because converting empty string to boolean returns false